src/main/kotlin/com/gamingbacklog/api/gamingbacklogapi/controllers/LibraryController.kt

```diff
@@ -12,12 +14,32 @@ import java.util.*
 @RestController
 @RequestMapping("/libraries")
 class LibraryController(private val libraryService: LibraryService) {
-  @GetMapping("/")
-  fun getLibraries(): ResponseEntity<List<Library>> {
+  @GetMapping("/all")
+  fun getAllLibraries(): ResponseEntity<List<Library>> {
     val libraries = libraryService.getAll()
     return ResponseEntity.ok(libraries)
   }
 
+  @CrossOrigin(origins = ["http://localhost:3000"])
+  @GetMapping("/")
+  fun getAllLibrariesWithGames(): ResponseEntity<List<LibraryResponse>> {
+    val libraries = libraryService.getAll()
+    // Needed for the frontend. Should refactor this and the one above + tests depending on what we need.
+    val librariesWithGames: List<LibraryResponse> = libraries.map { library -> convertLibraryToResponse(library) }
+    return ResponseEntity.ok(librariesWithGames)
+  }
+
+  private fun convertLibraryToResponse(library: Library): LibraryResponse {
```

Comment:
I think this should be in the LibraryService class, since that's where all logic happens while the controller is just the endpoints